### ISSUE TITLE
Cmus colors/appearance broken after update from 2.10.0-4 to 2.10.0-5

### ISSUE THREAD

**Issue by ViperZer0** [cmus/cmus#1338]
---

1) `uname -a`:
```
Linux viperzer0-desktop 6.6.34-1-lts #1 SMP PREEMPT_DYNAMIC Sun, 16 Jun 2024 14:45:31 +0000 x86_64 GNU/Linux
```

2) `cmus --version`:
```
cmus v2.10.0
```

3) `cmus --plugins`:
```
Input Plugins: /usr/lib/cmus/ip
  mad:
    Priority: 55
    File Types: mp3 mp2
    MIME Types: audio/mpeg audio/x-mp3 audio/x-mpeg
  modplug:
    Priority: 50
    File Types: mod s3m xm it 669 amf ams dbm dmf dsm far mdl med mtm okt ptm stm ult umx mt2 psm
    MIME Types:
  aac:
    Priority: 50
    File Types: aac
    MIME Types: audio/aac audio/aacp
  mpc:
    Priority: 50
    File Types: mpc mpp mp+
    MIME Types: audio/x-musepack
  wav:
    Priority: 50
    File Types: wav
    MIME Types:
  cue:
    Priority: 50
    File Types:
    MIME Types: application/x-cue
  flac:
    Priority: 50
    File Types: flac fla
    MIME Types:
  wavpack:
    Priority: 50
    File Types: wv
    MIME Types: audio/x-wavpack
  cdio:
    Priority: 50
    File Types:
    MIME Types: x-content/audio-cdda
  opus:
    Priority: 50
    File Types: opus
    MIME Types:
  vorbis:
    Priority: 50
    File Types: ogg oga ogx
    MIME Types: application/ogg audio/x-ogg

Output Plugins: /usr/lib/cmus/op
  pulse
  alsa
  oss
  jack
```

Upgraded my system last night and cmus now shows no colors other than the foreground text color. All the bars and panels are now completely black. In addition, cmus doesn't play nice if the terminal is resized. It looks like it still treats the terminal like it's full sized, so the lines wrap around and overflow the next available line, though I'm not sure if this is new or if it's always been that way. 

Here's what the window looks like now, I don't have a before screenshot. 

![image](https://github.com/cmus/cmus/assets/11594471/a24e3bf3-80d7-4892-b816-e248f769d7a2)
I did a cursory check to see if some other upgraded package broke cmus but I can't see anything. Neither my terminal nor ncurses has had a recent update, so the problem is probably with cmus?

**Comment by gavtroy** [comment]
It was probably recompiled with clang. The fix for this #1269 was included in cmus 2.11.0.

FYI the -5 is a packaging version. There's a downstream bug https://gitlab.archlinux.org/archlinux/packaging/packages/cmus/-/issues/1 although I don't have an account to reply there (ping: @christian-heusel).

**Comment by ViperZer0** [comment]
Yup, that looks like exactly the issue I'm dealing with. Thanks!

**Comment by christian-heusel** [comment]
Now fixed in the Arch Linux package (version 2.11.0-1), thanks a ton for bringing this issue to my attention, I wanted to look into this tomorrow 😊

**Comment by ViperZer0** [comment]
Wow thanks for getting it patched so quickly! Much appreciated!
